src/test/fuzz/crypto.cpp

```diff
@@ -63,12 +61,6 @@ FUZZ_TARGET(crypto)
                 (void)Hash(data);
                 (void)Hash160(data);
                 (void)sha512.Size();
-
-                if (fuzzed_data_provider.ConsumeBool()) {
-                    muhash *= MuHash3072(data);
```

Comment:
Nit: I don't think `operator*=(const MuHash3072&)` is exercised in `src/test/fuzz/muhash.cpp`. Perhaps worth expanding `src/test/fuzz/muhash.cpp` to keep coverage intact? :)

If doing so, consider fuzz covering the remaining currently non covered member functions of `MuHash3072`.

Reply:
thanks, done